mewarpx/mewarpx/mwxrun.py

```diff
@@ -54,6 +55,11 @@ class MEWarpXRun(object):
     def __init__(self):
         self.initialized = False
         self.simulation = picmi.Simulation(verbose=0)
+        self.comm = MPI.COMM_WORLD
```

Comment:
Do we actually need to define the communicator ourselves? It seems to me that if everything is handled with `installafterinit` calls appropriately this won't be necessary.

Reply:
Yep we don't need this anymore, I'll remove it.